packages/api/src/jobs/rss/refreshAllFeeds.ts

```diff
@@ -129,15 +132,15 @@ type QueuePriority = 'low' | 'high'
 export const queueRSSRefreshFeedJob = async (
   jobid: string,
   payload: any,
-  options = { priority: 'high' as QueuePriority }
```

Comment:
@jacksonh the default priority previously was `high` which was assigned to user added rss feed refresh job